### PR TITLE
Windows SDK dependency as NuGet packages

### PR DESCRIPTION
**Pull request by azchohfi**
Issue: #66 

## PR Type
What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?
Very hard to reference WinMDs.

## What is the new behavior?
No work needed to reference WinMDs, but to reference the NuGet Package.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../readme.md#supported)
- [ ] Pull Request has been submitted to the documentation repository [instructions](..\contributing.md#docs). Link: 
- [ ] Sample in sample app has been added / updated (for bug fixes / features)
    - [ ] Icon has been created (if new sample) following the [Thumbnail Style Guide and templates](https://github.com/windows-toolkit/WindowsCommunityToolkit-design-assets)
- [ ] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Header has been added to all new source files (run *build/UpdateHeaders.bat*)
- [ ] Contains **NO** breaking changes

### CODE REVIEW

**Comment by azchohfi** [comment]
We could probably remove _build/Install-WindowsSdkISO.ps1_ and _build/Install-WindowsSdk.ps1_

File: Microsoft.Toolkit.Sample.Wpf.App/Program.cs [hunk 1]
```diff
@@ -62,7 +62,7 @@ enum StartupKind
         [STAThread]
         public static void Main()
         {
-            var startupKind = StartupKind.Normal;
```
Comment (michael-hawker) [inline Microsoft.Toolkit.Sample.Wpf.App/Program.cs]: Was this the part you said wasn't supported anymore?  Should we have a comment here calling that this must be set this way and throw an exception in the else case below?

Comment (azchohfi) [reply Microsoft.Toolkit.Sample.Wpf.App/Program.cs]: We should remove the else clause. It doesn't work. There will be a future PR with a more complex scenario covered.

**Comment by azchohfi** [comment]
Let's validate it all on Monday.

**Comment by michael-hawker** [comment]
Nugets from PR worked like a charm on my Insider machine.  No large hassle with your instructions, just had one extra thing to do.  So, it's now:

1) Install the following
 - 18226 or later OS and SDK
 - .NET Core preview4
 - Suggested to install VS 2019 _Preview_
2) File New WPF .NET Core project
3) Add `Microsoft.Toolkit.Wpf.UI.XamlHost` package reference
4) In project, Add -> New Item -> **Application Manifest File**
5) Add following to compatibility/application section:
```
      <maxversiontested Id="10.0.18362.0"/>
      <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
```
6) Add following to application/windowSettings section:
```
      <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true/PM</dpiAware>
      <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitorV2, PerMonitor</dpiAwareness>
```
7) Create a Program.cs file with:
```
namespace WpfCoreTestApp
{
    public static class Program
    {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        public static void Main()
        {
            using (var xamlApp = new Microsoft.Toolkit.Win32.UI.XamlHost.XamlApplication())
            {
                var appOwnedWindowsXamlManager = xamlApp.WindowsXamlManager;

                var app = new App();
                app.InitializeComponent();
                app.Run();
            }
        }
    }
}
```
8) Edit the .csproj file and add the following to a PropertyGroup:
```
    <ApplicationManifest>app1.manifest</ApplicationManifest>
    <StartupObject>WpfCoreTestApp.Program</StartupObject>
```
9) Voila! XamlHost in your XAML:
```
<xamlhost:WindowsXamlHost InitialTypeName="Windows.UI.Xaml.Controls.TextBox" />
```
image

**Pull request merged**